code/modules/spells/spell_types/shadow_walk.dm

```diff
@@ -50,15 +50,19 @@
 	if(isspaceturf(newLoc))
 		to_chat(user, "<span class='warning'>It really would not be wise to go into space.</span>")
 		return
+	var/light_amount = newLoc.get_lumcount()
+	if(light_amount > SHADOW_SPECIES_LIGHT_THRESHOLD && is_blocked_turf(newLoc))
```

Comment:
```suggestion
	if(newLoc.get_lumcount() > SHADOW_SPECIES_LIGHT_THRESHOLD && is_blocked_turf(newLoc))
```
No reason to create a whole new local var here just for a single check.

Reply:
Yeah, I can definitely agree with that